cmd/zpool/zpool_main.c

```diff
@@ -2221,6 +2222,9 @@ zpool_do_import(int argc, char **argv)
 		case 'D':
 			do_destroyed = B_TRUE;
 			break;
+		case 'E':
+			do_imported = B_TRUE;
+			break;
```

Comment:
You'll want to add this option to the man page and comment above.  I'm surprised to see `E` was already part of the `getopt` string above.